boards/grand_central_m4/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+- Update to GPIO V2.
```

Comment:
nit: you could be more broad, since `atsamd-hal-0.15` required more than just GPIO v2

Reply:
Could you recommend some wording?

Reply:
How about something like

> - update to `atsamd-hal-0.15` (`v2` drivers of peripherals and removal of deprecated things)